Enterprise-CMCS/macpro-mdct-mcr
# MDCT-2496: MLR PDF a11y Improvements

### Description

Improving on a couple of items related to PDF accessibility:

- Marking PDF icons as decorative by setting `alt-text` attributes to null
- Matching headings hierarchy from Figma for screen reader traversal
- PDF metadata needs to be in the same hierarchy

### Related ticket(s)

MDCT-2496

---
### How to test

- Create a report
- Export it as PDF

Check for any `a11y` issues related to the description

### Important updates

N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
